app/react/Metadata/components/MetadataFormFields.js

```diff
@@ -356,8 +356,8 @@ export const mapStateToProps = (state, ownProps) => {
     attachments = selectedDocuments.size ? selectedDocuments.get(0).get('attachments') : undefined;
   }
 
-  if (storeKey === 'documentView') {
-    const entity = state.documentView.doc;
+  if (storeKey === 'documentView' || storeKey === 'documentViewer') {
```

Comment:
A quick search shows that there are actually no uses of `documentView` in the application. Most probably this was a typo and not an unhandled case.

Reply:
Removed